packages/conf-libopus/conf-libopus.1/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "2.0"
+maintainer: "https://github.com/ocaml/opam-repository/issues"
+homepage: "https://opus-codec.org/"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+authors: "Xiph.Org Foundation"
+license: "BSD"
```

Comment:
Same as the other, I think this needs to updated for spdx compliance

Reply:
Yes, I'm ready to execute the PR once this is merged.